apollo-router/src/plugins/telemetry/config_new/events.rs

```diff
@@ -244,49 +195,39 @@ where
     Attributes: Selectors<Request, Response, EventResponse> + Default,
     Sel: Selector<Request = Request, Response = Response> + Debug,
 {
-    pub(super) request: StandardEvent<Sel>,
-    pub(super) response: StandardEvent<Sel>,
-    pub(super) error: StandardEvent<Sel>,
+    pub(super) request: Option<StandardEvent<Sel>>,
+    pub(super) response: Option<StandardEvent<Sel>>,
+    pub(super) error: Option<StandardEvent<Sel>>,
```

Comment:
I normalize data here, previously we always had these values but internally they could have `EventLevel::Off` and this check was noisy.
So I removed `Off` from `EventLevel` and just set `None` instead.